### PR TITLE
fix: update 2-envs readme to template

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

File: 2-environments/README.md [hunk 1]
```diff
@@ -1,77 +1,204 @@
...
+</tr>
+</tbody>
+</table>
+
+For an overview of the architecture and the parts, see the
+[terraform-example-foundation README](https://github.com/terraform-google-modules/terraform-example-foundation).
+
+## Purpose
+
+The purpose of this step is to [...].
```
Comment (bharathkkb) [inline 2-environments/README.md:52]:
```suggestion
The purpose of this step is to setup development, non-production, and production environments within the Google Cloud organization that you've created.
```

File: 2-environments/README.md [hunk 2]
```diff
@@ -1,77 +1,204 @@
...
+href="https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks">3-networks</a></td>
+<td>Sets up base and restricted shared VPCs with default DNS, NAT (optional),
+Private Service networking, VPC service controls, on-premises Dedicated
+Interconnect, and baseline firewall rules for each environment. Also sets
+up the global DNS hub.</td>
+</tr>
+<tr>
+<td><a
+href="https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/4-projects">4-projects</a></td>
+<td>Set up a folder structure, projects, and application infrastructure pipeline for applications,
```
Comment (chris-ingrisani) [inline 2-environments/README.md:41]: Change to `Sets up` for consistency

File: 2-environments/README.md [hunk 3]
```diff
@@ -1,77 +1,204 @@
...
+   git push origin production
+   ```
+1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
+
+### Run Terraform locally
 
 1. Change into 2-environments folder.
 1. Run `cp ../build/tf-wrapper.sh .`
 1. Run `chmod 755 ./tf-wrapper.sh`.
 1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap. See any of the envs folder [README.md](./envs/production/README.md) files for additional information on the values in the `terraform.tfvars` file.
```
Comment (daniel-cit) [inline 2-environments/README.md:191]:
```suggestion
1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment and bootstrap. See any of the envs folder [README.md](./envs/production/README.md) files for additional information on the values in the `terraform.tfvars` file.
```

File: 2-environments/README.md [hunk 4]
```diff
@@ -1,77 +1,204 @@
...
+   ```
+   for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done
+   ```
+You can run `terraform output gcs_bucket_tfstate` in the 0-bootstap folder to obtain the bucket name.
 
 We will now deploy each of our environments(development/production/non-production) using this script.
 When using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for 2-environments step and only the corresponding environment is applied.
 
-To use the `validate` option of the `tf-wrapper.sh` script, the latest version of `terraform-validator` must be [installed](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-use-terraform-validator) in your system and in your `PATH`.
+To use the `validate` option of the `tf-wrapper.sh` script, the latest version of `terraform-validator` must be [installed](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-use-terraform-validator) in your system and in you `PATH`.
```
Comment (daniel-cit) [inline 2-environments/README.md:201]:
```suggestion
To use the `validate` option of the `tf-wrapper.sh` script, the latest version of `terraform-validator` must be [installed](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-use-terraform-validator) in your system and in your `PATH`.
```

Comment (daniel-cit) [inline 2-environments/README.md:201]:
```suggestion
To use the `validate` option of the `tf-wrapper.sh` script, the latest version of `terraform-validator` must be [installed](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-use-terraform-validator) in your system and in your `PATH`.
```

File: 2-environments/README.md [hunk 5]
```diff
@@ -1,77 +1,204 @@
...
-    1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
-1. Merge changes to production branch with `git checkout -b production` and `git push origin production`.
-    1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Review the plan output in your Master's web UI.
+1. Merge changes to development.
+   ```
+   git checkout -b development
+   git push origin development
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```
Comment (chris-ingrisani) [inline 2-environments/README.md:113]:
```suggestion
1. Review the apply output in your Master's web UI (you might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```

File: 2-environments/README.md [hunk 6]
```diff
@@ -1,77 +1,204 @@
...
+   git checkout -b non-production
+   git push origin non-production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to production branch.
+   ```
+   git checkout -b production
+   git push origin production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```
Comment (chris-ingrisani) [inline 2-environments/README.md:125]:
```suggestion
1. Review the apply output in your Master's web UI (you might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```

File: 2-environments/README.md [hunk 7]
```diff
@@ -1,77 +1,204 @@
...
+   git checkout -b development
+   git push origin development
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to non-production with.
+   ```
+   git checkout -b non-production
+   git push origin non-production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```
Comment (chris-ingrisani) [inline 2-environments/README.md:119]:
```suggestion
1. Review the apply output in your Master's web UI (you might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
```

**Comment by chris-ingrisani** [comment]
@bharathkkb besides the few minor requested changes, LGTM

File: 2-environments/README.md [hunk 8]
```diff
@@ -1,77 +1,204 @@
...
+<td>2-environments (this file)</td>
+<td>Sets up development, non-production, and production environments within the
+Google Cloud organization that you've created.</td>
+</tr>
+<tr>
+<td><a
+href="https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks">3-networks</a></td>
+<td>Sets up base and restricted shared VPCs with default DNS, NAT (optional),
+Private Service networking, VPC service controls, on-premises Dedicated
+Interconnect, and baseline firewall rules for each environment. Also sets
```
Comment (chris-ingrisani) [inline 2-environments/README.md:35]:
```suggestion
Interconnect, and baseline firewall rules for each environment. It also sets
```

**Pull request merged**